configuration/items.md

```diff
@@ -310,17 +313,67 @@ The following guidelines apply to user-added icon files:
 
 **Bitmaps or Vector Graphics:**
 openHAB can work with either Bitmap (`png`) or Vector (`svg`) icon files.
-The format should match the display capabilities of the user interfaces in use (e.g. Basic UI).
+The format should match the display capabilities of the user interfaces in use.
 It is thereby important to decide on one format beforehand; vector graphics are recommended.
-The setting can be changed via UI for most user interfaces.
+Most user interfaces don't expose this setting because they support both formats transparently.
+But the setting can be changed via UI in certain user interfaces.
 Please check the user interface documentation if in doubt.
 Note that image files with the wrong file ending will be ignored.
 
 Users may substitute their own icon for an icon from the default icon set by placing a file in the `$OPENHAB_CONF/icons/classic/` folder with the same filename as the name of the icon being substituted.
 
+#### Icon sources
+
+Some user interfaces supports other icons than those provided by openHAB.
+Thus the most generic reference to an icon is composed of 3 segments separated by a colon:
+
+1. The first segment is the icon source, e.g. "oh"
+
+2. The second segment is the icon set from this source, e.g. "classic"
```

Comment:
change 2. to 1. (this is why the linter complains)

Reply:
What do you mean?

Reply:
Ok I understood.

Reply:
Fixed